<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bigdataAnalyse</class>
 <widget class="QMainWindow" name="bigdataAnalyse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>872</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>BigData Messwert-Analyse</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QTextBrowser" name="terminalOut">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>150</y>
      <width>431</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="fileInfo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>331</width>
      <height>91</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Keine Daten eingelesen</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Messwert-Datei Einlesen</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>120</y>
      <width>671</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Datei-Informationen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Terminal-Ausgabe</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>771</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTextBrowser" name="filename">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="placeholderText">
        <string>Keine Datei ausgewählt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="browse">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Auswählen</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="TUlogo">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="workingdir.qrc">
      <normaloff>:/home/ian/Dokumente/Semester_7/Seminar 2.2/python/tu_logo.png</normaloff>:/home/ian/Dokumente/Semester_7/Seminar 2.2/python/tu_logo.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>170</width>
      <height>85</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="dataFilter">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>380</y>
      <width>101</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>anwenden</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>771</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>TechnoPhysicsRule Gmbh</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>331</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Rauheitsermittlung</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="mittenrauwert_ra">
       <property name="placeholderText">
        <string>Mittenrauwert Ra</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="quadratrauwert_rq">
       <property name="placeholderText">
        <string>Quadratische Rauheit Rq</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ra">
       <property name="text">
        <string>Rauheit berechnen</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>460</y>
      <width>16</width>
      <height>131</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>251</y>
      <width>227</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Datenfilter</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="radiomedfilt">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Medfilt</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="filter2">
         <property name="text">
          <string>Filter 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="filter2_2">
         <property name="text">
          <string>Filter 2</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>320</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Datenexport</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="saveData">
         <property name="text">
          <string>Daten speichern</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="savePlot">
         <property name="text">
          <string>Plot speichern</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="workingdir.qrc"/>
 </resources>
 <connections/>
</ui>
